doc/source/data/faq.rst

```diff
@@ -0,0 +1,308 @@
+   the trainers are bottlenecked on preprocessing and data loading.
+2. Slow, unpipelined, and/or unscalable preprocessing and ingestion pipelines for
+   **batch inference**, resulting in poor batch inference throughput and low GPU
+   utilization.
+
+In order to solve these problems without sacrificing usability, Ray Datasets simplifies
+parallel and pipelined data processing on Ray, providing a higher-level API while
+internally handling data batching, task parallelism and pipelining, and memory
+management.
+
+Who is using Ray Datasets?
+==========================
+
+To give an idea of Datasets use cases, we list a few notable users running Datasets
+integrations in production below:
+
+* Predibase is using Ray Datasets for ML ingest and batch inference in their OSS
+  declarative ML framework, `Ludwig <https://github.com/ludwig-ai/ludwig>`__, and
+  internally in their `AutoML product <https://predibase.com/>`__.
+* Amazon is using Ray Datasets for large-scale I/O in their scalable data catalog,
+  `DeltaCAT <https://github.com/ray-project/deltacat>`__.
+* Shopify is using Ray Datasets for ML ingest and batch inference in their ML platform,
+  `Merlin <https://shopify.engineering/merlin-shopify-machine-learning-platform>`__.
+* Ray Datasets is used as the data processing engine for the 
+  `Ray-based Apache Beam runner <https://github.com/ray-project/ray_beam_runner>`__.
+* Ray Datasets is used as the preprocessing and batch inference engine for
+  :ref:`Ray AIR <air>`.
+
+
+If you're using Ray Datasets, please let us know about your experience on the
+`Slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`__  or
+`Discourse <https://discuss.ray.io/>`__; we'd love to hear from you!
+
+What should I use Ray Datasets for?
+===================================
+
+Ray Datasets is the standard way to load, process, and exchange data in Ray libraries
+and applications, with a particular emphasis on ease-of-use, performance, and
+scalability in both data size and cluster size. Within that, Datasets is designed for
+two core uses cases:
+
+* **ML (training) ingest:** Loading, preprocessing, and ingesting data into one or more
+  (possibly distributed) model trainers.
+* **batch inference:** Loading, preprocessing, and performing parallel batch
+  inference on data.
+
+We have designed the Datasets APIs, data model, execution model, and
+integrations with these use cases in mind, and have captured these use cases in
+large-scale nightly tests to ensure that we're hitting our scalability, performance,
+and efficiency marks for these use cases.
+
+See our :ref:`ML preprocessing docs <datasets-ml-preprocessing>` for more information on
+this positioning.
+
+What should I not use Ray Datasets for?
+=======================================
+
+Ray Datasets is not meant to be used for generic ETL pipelines (like Spark) or
+scalable data science (like Dask, Modin, or Mars). Datasets integrates with these
```

Comment:
maybe this is too much to ask here, but naively speaking what's the notable diff between data science as done by e.g. Dask and our above use case of "loading, prep. and ingesting data for ML"? What exactly are we missing in this positioning? I'd rather be very clear about this here.